docs/reference/api/platform/Logger.md

```diff
@@ -0,0 +1,164 @@
+## Logger
```

Comment:
Can you please add information on how big the buffer size is and how it can be configured?